addons/im_livechat/controllers/main.py

```diff
@@ -124,7 +124,7 @@ def get_session(self, channel_id, anonymous_name, previous_operator_id=None, cha
                     "steps": welcome_steps.mapped(lambda s: {"scriptStep": s.id}),
                 }
                 store.add(chatbot_script)
-                store.add(welcome_steps)
+                store.add(welcome_steps, welcome_steps._to_store_defaults() + ["message"])
```

Comment:
I believe the main goal was removing the transformation, not to move it out of default.